wifiphisher/common/phishingpage.py

```diff
@@ -400,6 +403,10 @@ def add_user_templates(self):
             local_template = PhishingTemplate(template)
             self._templates[template] = local_template
 
+    @property
+    def template_directory(self):
```

Comment:
Is there a reason for this property here?

Reply:
as an getter for the _template_directory private field